docker/.env.example.full

```
@@ -467,6 +463,16 @@ ARK_API_KEY=""
 # Secret value
 ARK_SECRET_ACCESS_KEY=""
 
+# Enable external ask providers (dify/coze). Default false.
+# (Optional - default: False)
+# Type: bool
+ASK_PROVIDER_ENABLED="False"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)